bridge/client/app/_views/ktb-settings-view/ktb-integration-view/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.spec.ts

```diff
@@ -320,6 +328,66 @@ describe('KtbModifyUniformSubscriptionComponent', () => {
     });
   });
 
+  it('should not show a notification when the component is initialized', () => {
+    // given
+    fixture.detectChanges();
+
+    // then
+    // Has to be retrieved by document, as it is not created at component level
+    const notifications = document.getElementsByTagName('dt-confirmation-dialog-state');
+    expect(component.unsavedDialogState).toBeNull();
+    expect(notifications.length).toEqual(0);
+  });
+
+  it('should not allow navigation for unsaved changes', () => {
+    // given
+
+    // when
+    component.webhookFormDirty = true;
+    fixture.detectChanges();
+
+    // then
+    expect(component.canDeactivate()).not.toEqual(true);
+  });
+
+  it('should show a dialog when showNotification is called', () => {
+    // given
+    component.webhookFormDirty = true;
+    fixture.detectChanges();
+
+    // when
+    component.showNotification();
+    fixture.detectChanges();
+
+    // then
+    const notification = document.getElementsByTagName('dt-confirmation-dialog-state');
+    expect(notification.length).toEqual(1);
+
+    // We have to reset the state, as the dt-confirmation-dialog component has some pending timer open
+    // and the test will not complete
+    component.hideNotification();
+    fixture.detectChanges();
+  });
+
+  it('should not show dialog when the notification was closed', () => {
+    // given
+    component.webhookFormDirty = true;
+    fixture.detectChanges();
+
+    // given
+    component.showNotification();
+    fixture.detectChanges();
+
+    // when
+    component.hideNotification();
+    fixture.detectChanges();
+
+    // then
+    const notification = document.getElementsByTagName('dt-confirmation-dialog-state')[0];
+    // It still exists in the dom but is hidden - so we test for aria-hidden
+    expect(notification.getAttribute('aria-hidden')).toEqual('true');
+  });
```

Comment:
They should also be moved to Cypress. Notification checks can be re-used from `BasePage.ts`.